cocos/2d/CCLabel.cpp

```diff
@@ -1405,7 +1405,7 @@ void Label::onDraw(const Mat4& transform, bool transformUpdated)
             //draw text without outline
             glprogram->setUniformLocationWith4f(_uniformEffectColor,
                 _effectColorF.r, _effectColorF.g, _effectColorF.b, 0.f);
-            break;
```

Comment:
@animaseed  if you return here, the text without outline won't be drawn... @WenhaiLin Thoughts?